interface/backend/api/views.py

```diff
@@ -37,16 +41,34 @@ class ImageAvailableApiView(APIView):
 
     def get(self, request):
         """
-        Return a list of files and folders in dataset in the form
+        Return a sorted(by name) list of files and folders 
+        in dataset in the form
         {'directories': [
             {
                 'name': directory_name1,
                 'children': [ file_name1, file_name2, ... ]
             }, ... ]
         }
-
+        
+        TODO Dynamically fetch deeper directories
         """
-        return JsonResponse({'directories': []})
+        # example source:
+        # /images/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178
+        src_dir = os.environ.get('IMAGE_SOURCE', '/images')
+        fss = FileSystemStorage(src_dir)
+        list_dirs = fss.listdir(src_dir)
+        dirs = sorted(list_dirs[0])
+        tree = []
+        if dirs:
+            for dirname in dirs:
+                dir_contents = fss.listdir(os.path.join(src_dir, dirname))
+                filenames = sorted(dir_contents[1])
+                dir_node = {
+                    'name': dirname,
+                    'children': filenames
```

Comment:
(Very picky mode activated but whilst I'm commenting.. please add trailing commas as it prevents diff noise if someone adds another row here later)